### DISCUSSION TITLE
An example on how one can use this toolkit

### DISCUSSION THREAD

**Discussion by MuhammadPen** [readium/ts-toolkit#17] [General]
It would be helpful if there is a simple code example on using this toolkit. As a new comer to readium and someone who just wants to add readium to his react app, this is quite confusing.

**Comment by HadrienGardeur** [comment]
Hello @MuhammadPen,

Readium Web is still in its infancy and will eventually contain a test app and full documentation but for now the team is mostly focused on building various navigators (see https://readium.org/architecture):

- the one for EPUB reflow is in active development (see the list of PR)
- PDF is next, with work planned in October/November
- and EPUB FXL is planned for early next year

If you'd like to contribute and help us build the project, we recommend joining our bi-monthly calls, that's the best way to get introduced to the team and project.
If you're interested in simply using Readium Web to build a Web Reader, you'll have to bear with us right now as this is not ready for primetime yet.


There are other options that you can look into in the meantime:

- if you're looking to build a React app for desktop/laptops using Electron, you can take a look at Readium Desktop
- if you're looking to build a React native app for mobile, we recommend using Readium Mobile

**Reply by dysbulic** [reply]
> If you'd like to contribute and help us build the project, we recommend joining our bi-monthly calls, that's the best way to get introduced to the team and project.

So, as per [readium.org/development/contributing/](//readium.org/development/contributing/), I tried filling out the licensing agreement and emailing it to secretary@readium.org with a question about when your meetups are, but it bounced. Could I get some more info?

My time is really limited at the moment, but one of my projects is a collaborative filesystem to be used as a distributed proofreading system. I could really use a solid EPUB reader and want to get a better sense of where y'all're at.

**Comment by MuhammadPen** [comment]
Appreciate the quick response, my use case was a react web app with some customizations around the reading experience. I guess I have to wait for readium-web to mature. Until then, do you have any good and stable recommendation for an alternative? I looked at epub.js but I am not sure if its as feature rich.

**Reply by HadrienGardeur** [reply]
epub.js is probably more feature rich right now, but it's not targeting PDF and audiobooks as well.

Readium Web should be more stable in late Q1 2023 if you'd like to explore it in a less experimental stage. On our side at De Marque, we've started deploying an early version of Readium Web, which will provide us a lot of useful feedback to improve the toolkit.

**Comment by MuhammadPen** [comment]
I ended up using [r2d2bc](https://github.com/d-i-t-a/R2D2BC)
